### ISSUE TITLE
where are γind and β ind in code? 

### ISSUE THREAD

**Issue by mzprose** [Aaditya-Singh/SAFIN#6]
How is this parameter calculated in the code in the paper, and which parts of the code?

**Comment by Aaditya-Singh** [comment]
They are referred to as `shared_weight` and `shared_bias` in the `SAFIN` class. Refer to this part of the code for further details: https://github.com/Aaditya-Singh/SAFIN/blob/338ae06eb257b83f41d2dfea7a553fec29cfe0be/function.py#L91

**Comment by mzprose** [comment]
Thank you for your reply. Excuse me, the share_weight and share_bias are defined in SAFIN, and they are initialized to 1, 0 in the code. How do you change these values in the subsequent training and learning process?

**Comment by Aaditya-Singh** [comment]
This is the backpropagation step in the code, if that's what you meant: 
https://github.com/Aaditya-Singh/SAFIN/blob/main/train.py#L164

I would recommend checking out the tutorials if you are new to Pytorch: https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html

**Comment by mzprose** [comment]
Sorry, I said that there may be a problem. My question is, your γ ind and β ind are only defined in SAFIN block,  and then learns them during the backpropagation process. Didn't other blocks call these two parameters?

**Comment by Aaditya-Singh** [comment]
All of the normalization parameters are defined within the `SAFIN` class in the code which is consistent with Equation (1) in the paper. Kindly refer to our paper and the code for more details.

**Comment by mzprose** [comment]
OK, thanks for your reply!
